room_assistance/indico_room_assistance/plugin.py

```diff
@@ -0,0 +1,94 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2019 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from flask import session
+from flask_pluginengine import render_plugin_template, url_for_plugin
+
+from indico.core import signals
+from indico.core.plugins import IndicoPlugin
+from indico.core.settings.converters import ModelListConverter
+from indico.modules.rb.models.rooms import Room
+from indico.util.string import natural_sort_key
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.fields import EmailListField, IndicoQuerySelectMultipleField, PrincipalListField
+from indico.web.menu import TopMenuItem
+
+from indico_room_assistance import _
+from indico_room_assistance.blueprint import blueprint
+from indico_room_assistance.definition import RoomAssistanceRequest
+from indico_room_assistance.util import is_room_assistance_support
+
+
+def _order_func(object_list):
+    return sorted(object_list, key=lambda r: natural_sort_key(r[1].full_name))
+
+
+class RoomAssistanceForm(IndicoForm):
+    _fieldsets = [
+        ('Startup assistance emails', ['room_assistance_recipients', 'rooms_with_assistance',
+                                       'room_assistance_support']),
+    ]
+
+    room_assistance_recipients = EmailListField(_('Recipients'))
```

Comment:
I think it would make sense to add a description of this field as there must be one email address per line, no?
Something like "Notifications about room assistance requests are sent to these email addresses (one per line)"?

Reply:
Seems pretty obvious to me. Anyway, only we configure this plugin.

Reply:
At first I would have added a list of email addresses separated by commas.

Reply:
Yeah, but if that is configured by us, then it is not mandatory. However, it deosn't harm to add it ;) I will add a short description.